schema/deploy/shipping/views.sql

```diff
@@ -727,7 +727,6 @@ create or replace view shipping.sample_with_best_available_encounter_data_v1 as
   left join samples_with_manifest_data using (sample_id)
   left join site_details on (site_manifest_details similar to manifest_regex)
   left join warehouse.site on (samples_with_manifest_data.site_id = site.site_id)
-  where sample.identifier is not null
```

Comment:
I finally found [the context for this filter](https://github.com/seattleflu/id3c-customizations/pull/33#pullrequestreview-361337238). You suggested adding a collection identifier column and it looks like I didn't totally get it then. But I think that might be nice to have now.

Reply:
Ohhhhhhh, right. I can add collection identifier to this view then.

Reply:
Adding a column means we'd have to bump the version number right?

Reply:
No, only taking away existing columns or substantially changing what's being returned would warrant a version bump.

Reply:
But adding a column breaks the REVERT script...I think this is why I'm having so much trouble with when to bump the version number

Reply:
Oh, right. There is the weird workaround I've been doing in the revert script where I drop then re-create the view afterward. But then we may also have to drop the views that are dependent on this view before re-creating it.

I think it can technically be done without bumping the version number, but it does require more `revert` script manipulation than we'd typically like to do.

Reply:
I'd say don't worry about adding `collection_identifier` to `shipping.sample_with_best_available_encounter_data_v1` for now. It has `sample_id`, which is what matters most.

You may know this, but in case not: the reason the revert script needs to be updated is because `create or replace view` only works if you don't change the name, type, or order of any existing columns. So with `create or replace view` you can add new columns after the existing columns or change the values in an existing column (using the same name and type), but to insert a new column amidst the existing columns, you must do `drop + create`, which then has to cascade to dependent views. One pattern for managing this, which we're not doing yet, is to always drop all views at the top of the file and then issue the creates, i.e. drop+recreate on every deploy even if you technically don't need to.